MAINTAINERS.yml

```diff
@@ -1834,6 +1834,21 @@ Nuvoton Numicro Platforms:
   labels:
     - "platform: Nuvoton Numicro"
 
+Raspberry Pi Pico Platforms:
+  status: maintained
+  maintainers:
+    - yonsch
+  files:
+    - boards/arm/rpi_pico/
+    - boards/arm/adafruit_kb2040/
+    - boards/arm/sparkfun_pro_micro_rp2040/
+    - dts/arm/rpi_pico/
+    - dts/bindings/*/raspberrypi,pico*
```

Comment:
The MAINTAINERS parser is unhappy because this matches a directory unfortunately: `drivers/sensor/rpi_pico_temp/`. I think you may have to break it down, for example for stm32 I see

```
    - drivers/*/*stm32*/
    - drivers/*/*stm32*.c
    - drivers/*/*stm32*.h
```

Reply:
Notice that you can run that check locally with `./scripts/ci/check_compliance.py -m MaintainersFormat` (runs on the latest commit by default)